R/verb-window.R

```diff
@@ -1,5 +1,24 @@
 #' Override window order and frame
 #'
+#' @description
+#' `window_order()` sets the order for subsequent invocations
+#' of window functions on a lazy table.
+#' The order is active only for the next verb.
```

Comment:
Is that also true for `window_frame()`? 

Maybe it's easier to just leave the `window_order()` persistence as is?

Reply:
`window_frame()` is different because it never stacks. I agree we should remove it there too: #475.

Reply:
If we leave `window_order()` persistence, we need to provide a way to reset it. This also leaves us vulnerable to `. %>% window_order(a) %>% select(-a)`.

I suggest we make a clean cut here, perhaps after the dplyr CRAN release.

Reply:
But clearly `window_order()` and `window_frame()` need to persist the status of the other. That's starting to seem rather complicated to me.

Reply:
I don't mind persisting the status of the order in `window_order()` if we provide a way to reset it. What would be a good API -- `window_order(.add = FALSE)` ?

Reply:
`window_order()` and `window_frame()` now respect each other, added tests. `window_order()` still overrides the previously set order.